src/test/resources/features/refapp/06-patient/EditPatientRelationship.feature

```diff
@@ -0,0 +1,2 @@
+Feature: EditRelationshipType
```

Comment:
Did you just forget to write the user scenarios in the file

Reply:
This is work in Progress thanks